content/gherkin/reference.md

```diff
@@ -330,6 +331,60 @@ to match the step against a step definition.
 
 You can also use parameters in [multiline step arguments](#step-arguments).
 
+## Rule
+
+The `Rule` entity is a rather new idiom in Gherkin (since Gherkin v6).
+Each `Rule` is intended to represent one *business rule* that should be implemented.
+A `Rule` is used to group together several `Example` (aka: `Scenario`) / `Scenario Outline`
+that belong to this *business rule*. In addition, a `Rule` may contain a `Background` section.
+
+Rule parent:
+
+- `Feature` (cardinality: one)
```

Comment:
What do you mean with "(cardinality: one)"? A `Rule` can belong to only one `Feature`?

Reply:
Exactly, a one-to-one relationship has the cardinality "one" (in the direction towards the `Feature`). Cardinality "one" means if a `Rule` is  used, then the Rule must always be below a `Feature`.

I assume, you know what an UML class diagram is ?!?
(and the cardinality for a relation between classes or objects).